atomic_defi_design/qml/Components/DexModalHeader.qml

```diff
@@ -0,0 +1,24 @@
+import QtQuick 2.15
+
+DexRectangle {
+	id: _headerBackground
+	property alias text: _title.text
+	/*width: parent.width
```

Comment:
If you commented some code and want to keep it, you may want to add a statement which says why you commented it. If you have nothing to say about it, it is probably useless to keep it.